src/main/java/de/tum/in/www1/artemis/domain/iris/settings/IrisLectureIngestionSubSettings.java

```diff
@@ -0,0 +1,16 @@
+package de.tum.in.www1.artemis.domain.iris.settings;
+
+import jakarta.persistence.DiscriminatorValue;
+import jakarta.persistence.Entity;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+/**
+ * Represents the specific ingestion sub-settings of lectures for IRIS.
+ * This class extends {@link IrisSubSettings} to provide specialized settings required for lecture data ingestion.
+ */
+@Entity
+@DiscriminatorValue("ingestion")
```

Comment:
```suggestion
@DiscriminatorValue("INGESTION")
```

Reply:
Or even better "LECTURE_INGESTION" as we will have other types of ingestion in the future.